Practice1/src/css/readhealth.css

```diff
@@ -0,0 +1,87 @@
+.page-read-health {
+    position: relative;
+    width: 100%;
+    padding-bottom: 144px;
+}
+
+.page-read-health-img {
+    position: absolute;
+    z-index: -1;
+    width: 100%;
+    height: 100%;
+}
+
+.page-read-health-heading {
+    padding-top: 50px;
+    color: var(--white);
+    width: 100%;
+    text-align: center;
+    font-size: 40px;
+}
+
+.page-read-health-container {
+    margin-top: 48px;
+    display: flex;
+    justify-content: center;
+}
+
+.mask {
+    margin: 50px 40px 0px 61px;
+    width: 302px;
+    border: 0px;
+    /* box-sizing: border-box; */
+    display: flex;
+    flex-direction: column;
+}
+
+.mask .mask-img {
+    width: 302px;
+    vertical-align: middle; 
+}
+
+.page-read-health-blog {
+    background-color: var(--white);
+}
+
+/* .top-medical {
+    font-size: 22px;
+    text-align: center;
+    margin-top: 0px;
+} */
+
+.top-medical-input {
+    font-size: 22px;
+    padding-top: 11px;
+    text-align: center;
+    margin-top: 0px;
+}
+
+.lorem {
+    /* position: relative; */
+    /* padding-top: 7px; */
+}
+
+.lorem-input {
+    font-size: 13px;
+    text-align: center;
+    line-height: 15px;
+    letter-spacing: 0.0200893em;
+    text-align: justify;
+    color: var(--gray);
+    padding: 7px 12px 0 13px;
+}
+
+.page-read-health-footer {
+    position: relative;
+    padding: 7px 0px 12px 0px;
+    display: flex;
+    justify-content: space-around;
+}
+
+.page-read-health-footer .input-left {
+    margin-right: 114px;
+}
+
+.page-read-health-footer .input-right {
```

Comment:
When naming class, we should avoid position words such as left, right, bottom, and top because if clients change their requirement to move the input from the right to the left, we have to rename the class to match the new position, which isn't a good idea.

Reply:
Fixed